### ISSUE TITLE
DetectEntireAnomalyBySrCnn - No anomalies detected as of version 1.5.4 or greater

### ISSUE THREAD

**Issue by luisquintanilla** [dotnet/machinelearning#5891]
**System Information (please complete the following information):**
 - OS & Version: Windows 10
 - ML.NET Version: ML.NET 1.5.4 and greater
 - .NET Version: .NET 5

**Describe the bug**

When using `DetectEntireAnomalyBySrCnn` with versions 1.5.4 or greater, no anomalies are detected when running the `DetectAnomaly` method from the [phone call tutorial](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/phone-calls-anomaly-detection).

Changing options does nothing. Has anything changed with this algorithm that would suddenly make it stop working after 1.5.4?

If you run the sample using version 1.5.2 with the same data and options it works.

Related docs issue dotnet/docs#23888

**To Reproduce**

Follow the steps in the [phone call tutorial](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/phone-calls-anomaly-detection). For reference, here's a [completed code sample](https://github.com/dotnet/docs/tree/main/docs/machine-learning/tutorials/snippets/phone-calls-anomaly-detection/csharp)

**Expected behavior**

Anomalies are detected.

**Comment by michaelgsharp** [comment]
@luisquintanilla since this is a regression I would say its probably a higher priority. @briacht thoughts?

**Comment by briacht** [comment]
Yeah if the algorithm isn't working anymore we should definitely try and fix that

**Comment by michaelgsharp** [comment]
This is the PR that caused this bug https://github.com/dotnet/machinelearning/pull/5202.

**Comment by matouskozak** [comment]
The current tests use sensitivity 87 nor 64 (`test/Microsoft.ML.TimeSeries.Tests/TimeSeriesDirectApi.cs`)

This  sample program should be updated https://github.com/dotnet/samples/blob/main/machine-learning/tutorials/PhoneCallsAnomalyDetection/Program.cs

The code itself was fixed in https://github.com/dotnet/machinelearning/pull/7610
